libr/util/d/meson.build

```diff
@@ -0,0 +1,17 @@
+sdb_files = [
+  'pokered'
+]
+
+foreach file : sdb_files
+  outfile = '@0@.sdb'.format(file)
+  infiles = ['@0@.sdb.txt'.format(file)]
+  custom_target(outfile,
+    input: infiles,
+    output: outfile,
+    command: sdb_gen_cmd,
+    depends: sdb_exe,
+    build_by_default: true,
+    install: true,
+    install_dir: join_paths(r2_sdb, 'util/d')
```

Comment:
I think this is not good.
1) you should use `join_paths (r2_sdb, 'util', 'd')`
2) util/d is not a good name. If you see the other similar cases, they *are not* named `bins/d/`, `anal/d`, etc., but they usually have a proper name. In this case, maybe `charset` would make more sense, or something like that.

Reply:
what do you think of util/charset instead?

Reply:
Or util/d/charset @ret2libc ?

Reply:
@trufae why the `util/d`? If you look at the other sdb files we install, they are installed in things like `fcnsign`, `format`, `opcodes`, etc. and not in `anal/fcnsign`, `bin/format`, etc. So to be consistent I would just use `charset`.

Reply:
I agree with @ret2libc . It seem more meaningful to put in charset. I will have to modify code.

Just remember that for the moment no output is created with the sdb files with meson. It is my priority.